lib/yjit-metrics/report_types/bloggable_speed_report.rb

```diff
@@ -52,14 +52,15 @@ def look_up_data_by_ruby(only_platforms: YJITMetrics::PLATFORMS, in_runs: false)
 
         # Order matters here - we push No-JIT, then MJIT(s), then YJIT and finally TruffleRuby when present
         @configs_with_human_names = [
-            ["No JIT", @no_jit_config],
-        ]
-        # TODO: get the ruby version from the description in the metadata
-        @configs_with_human_names.push(["MJIT3.0", @with_mjit30_config]) if @with_mjit30_config
-        @configs_with_human_names.push(["MJIT", @with_mjit_latest_config]) if @with_mjit_latest_config
-        @configs_with_human_names.push(["YJIT 3.3", @with_prev_yjit_config]) if @with_prev_yjit_config
-        @configs_with_human_names.push(["YJIT 3.4dev", @with_yjit_config])
-        @configs_with_human_names.push(["Truffle", @truffle_config]) if @truffle_config
+          ["No JIT", @no_jit_config],
+          ["MJIT3.0", @with_mjit30_config],
+          ["MJIT", @with_mjit_latest_config],
```

Comment:
Do we need to keep them?

Reply:
We generally bisect commits after the last release (to find a regression point in an under-development version), so it seems fine to have graphs only for dates without MJIT. It should at least cover the past one year.

Having said that, since this only helps maintainability, it seems fine to leave it for now if it's hard to clean them up.